scripts/make_fake_pm_data.js

```diff
@@ -0,0 +1,134 @@
+const async = require("async");
+const fs = require("fs");
+const _ = require("underscore");
+const json2csv = require("json2csv");
+const argv = process.argv.slice(2);
+const BUILDINGS = require("./" + argv[0]);
+
+function randomNumber(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+}
+
+function randomNumberOfStandardDeviations(min, max) {
+    return ((Math.random() * (max - min + 1) + min))
+}
+
+function shiftCoordinates(coordinate, shiftValue) {
+    return (coordinate - shiftValue);
+}
+
+function standardDeviation(mean, dataArray, numberOfEntries) {
+    let sumDistanceFromMean = 0 
+    dataArray.map(data => {
+        sumDistanceFromMean += Math.pow((data - mean), 2);
+    })
+
+    let stdDev = Math.round(Math.sqrt((sumDistanceFromMean / numberOfEntries)))
+    return stdDev;
+}
+
+function initialRead(cb) {
+    console.log("Reading data...")
+    const buildingJSON = BUILDINGS.map(building => {
+        return building
+    })
+    cb(null, buildingJSON);
+}
+
+function generateBuildingObject(buildingJSON, cb) {
```

Comment:
The param isn't actually JSON (type string) anymore. It's a JS array, so the param name is misleading... I'd expect within the function body to have to do `JSON.parse()` given the name but you haven't.